packages/devtools_shared/lib/src/memory/adb_memory_info.dart

```diff
@@ -3,8 +3,10 @@
 // found in the LICENSE file.
 
 // TODO(terry): Need the iOS version of this data.
+import '../../devtools_shared.dart';
```

Comment:
import the library you need directly instead of the library that exports other libraries